static/src/javascripts/projects/journalism/snippet-feedback.js

```diff
@@ -10,6 +10,28 @@ const SnippetFeedback = (options: { scroll: boolean } = { scroll: true }) => {
     let snippets = [...document.querySelectorAll('.explainer-snippet--new')];
     const variant = getAssignedVariant(SnippetFourVariants);
 
+    const sendOldStyleInteraction = (snippetId, component, value) => {
+        ophan.record({
+            atomId: snippetId,
+            component,
+            value,
+        });
+    };
+
+    const sendNewStyleInteraction = (snippetId, snippetType, action) => {
+        ophan.record({
```

Comment:
You will get type check by using [`submitComponentEvent`](https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/acquisitions-ophan.js) instead.

Reply:
ah, thanks!